packages/internal/src/paths.ts

```diff
@@ -188,7 +190,7 @@ export const processPagesDir = (
   return pagePaths.map((pagePath) => {
     const p = path.parse(pagePath)
 
-    const importName = p.dir.replace(path.sep, '')
+    const importName = p.dir.replace('/', '')
```

Comment:
I've had different results when it comes to paths on different shells on Windows before. Where have you tested your fix? I usually try to test everything in Cmd, PowerShell and git-bash.

Reply:
Interesting ... i did not know that.  I was using PowerShell, but just ran yarn test from Cmd and git-bash as well.  They appear to all work.  Let me know if I miss something!

My testing is the actual app (using the framework) was through brute force as I still haven't got the rsync copy working yet.  The information you've posted was super helpful, but something's still not quite right yet.  I'll work on that more another day before I bug you or anyone else about that.